fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/DirtyContextBeforeAndAfterClassTestExecutionListener.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.integration.http.api;
+
+import org.springframework.core.Ordered;
+import org.springframework.test.annotation.DirtiesContext.HierarchyMode;
+import org.springframework.test.context.TestContext;
+import org.springframework.test.context.support.AbstractTestExecutionListener;
+
+/**
+ * @author Sam Brannen
```

Comment:
Who is `Sam Brannen`? I do not believe we have his CLA on-file.

Reply:
I wasn't sure what to do with the author thing there. It was basically copy and pasted from stackoverflow so i didn't really want to take credit.

Reply:
I could switch the author to myself and reference the original in the javadoc, whatever you prefer

Reply:
Maybe you as the author, and link to where you found the code.